<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1737</width>
    <height>970</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
font: 75 11pt &quot;Ubuntu&quot;;
background-color: rgb(170, 230, 235)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>931</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Экран!!!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>470</y>
      <width>921</width>
      <height>131</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="Button1">
     <property name="styleSheet">
      <string notr="true">color: rgb(50, 50, 50);
border-radius:  80px;                     
border: 2px solid #094065;
 
background-color: rgb(37, 246, 255);

background-color: rgb(16, 188, 232);</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button2">
     <property name="styleSheet">
      <string notr="true">border-radius:  80px;                     
border: 2px solid #094065;
 
background-color: rgb(37, 246, 255);

background-color: rgb(16, 188, 232);</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button3">
     <property name="styleSheet">
      <string notr="true">border-radius:  80px;                     
border: 2px solid #094065;
 
background-color: rgb(37, 246, 255);

background-color: rgb(16, 188, 232);</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button4">
     <property name="styleSheet">
      <string notr="true">border-radius:  80px;                     
border: 2px solid #094065;
 
background-color: rgb(37, 246, 255);

background-color: rgb(16, 188, 232);</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button5">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:  80px;                     
border: 2px solid #094065;
 
background-color: rgb(37, 246, 255);

background-color: rgb(16, 188, 232);</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>460</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(50, 50, 50);
border-radius:  80px;                     
border: 2px solid #094065;
 
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/power.png</normaloff>../images/power.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>101</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="homeButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(50, 50, 50);
border-radius:  80px;                     
border: 2px solid #094065;
 background-color: rgb(254, 254, 254);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/home.jpg</normaloff>../images/home.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>107</width>
      <height>102</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="lightButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(50, 50, 50);
border-radius:  80px;                     
border: 2px solid #094065;
 
background-color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/light.png</normaloff>../images/light.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>105</width>
      <height>89</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>91</width>
      <height>231</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="rightButton">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>220</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>41</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>220</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/arrow.png</normaloff>images/arrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>148</width>
      <height>117</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1737</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string/>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>on_pushButton_clicked()</signal>
 </slots>
</ui>
